app/pods/publications/index/template.hbs

```diff
@@ -30,7 +30,7 @@
         {{/if}}
 
         {{#if this.filterTableColumnOptionKeys.numacNumberFilterOption}}
-          <th>{{t "numac-number"}}</th>
+          <th>{{t "publications-table-numacnummer-bs"}}</th>
```

Comment:
meerdere keys voor numac-nummer, deze was duidelijker om dat de gebruikte plaats bij in de naam staat)